### ISSUE TITLE
integrated terminal "search" breaks clipboard on linux

### ISSUE THREAD

**Issue by farfromrefug** [VSCodium/vscodium#1896]
Type: <b>Bug</b>

* open an integrated terminal
* search for text in the terminal and leave the search box open
* open another app and try to work with the clipboard

=> whatever you try to copy in all apps (including VSCode). it will always paste the current content of the terminal search box
* close the terminal search box
=> clipboard works again in all apps (including VSCode)

VSCodium version: VSCodium 1.88.1 (0d01a72525276e5e3771fe5851b9c83de0de1e2b, 2024-04-13T01:53:59.491Z)
OS version: Linux x64 6.8.0-76060800daily20240311-generic
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|12th Gen Intel(R) Core(TM) i7-12700H (20 x 2300)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|7, 7, 5|
|Memory (System)|62.64GB (25.94GB free)|
|Process Argv|--unity-launch|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|pop|
|XDG_CURRENT_DESKTOP|Unity|
|XDG_SESSION_DESKTOP|pop|
|XDG_SESSION_TYPE|x11|
</details><details><summary>Extensions (53)</summary>

Extension|Author (truncated)|Version
---|---|---
better-comments|aar|3.0.2
dashyeah|bob|1.0.3
turbo-console-log|Cha|2.10.4
source-map-visualization|col|2.2.0
font-preview|ctc|2.2.1
vscode-eslint|dba|2.4.4
vscode-dash|dee|2.4.0
emulate|Die|1.6.0
deeplink|ema|0.0.10
json-tools|eri|1.0.2
prettier-vscode|esb|10.4.0
vscode-open-in-github|fab|2.3.0
vscode-svelte-snippets|fiv|0.5.0
code-runner|for|0.12.2
incrementor-henrikvilhelmberglund|hen|2.4.1
npm-dependency-links|her|1.2.0
vscode-postfix-ts|ipa|1.13.2
svg|joc|1.5.3
vscode-map-preview|jum|0.6.0
i18n-ally|lok|2.12.0
materialdesignicons-intellisense|luk|4.1.0
json-to-ts|Mar|1.8.0
hex-hover-converter|maz|1.2.2
template-string-converter|meg|0.6.1
isort|ms-|2023.10.1
python|ms-|2024.4.1
vscode-pylance|ms-|2024.5.1
jupyter|ms-|2024.3.1
java-prettier-formatter|mwp|0.1.6
gradle-language|nac|0.2.3
color-highlight|nau|2.8.0
vs-qalc|nor|0.4.0
vscode-ios-common-files|Ort|1.0.5
vscode-versionlens|pfl|1.9.2
vscode-thunder-client|ran|2.23.0
vscode-xml|red|0.26.1
vscode-yaml|red|1.14.0
opencv-image|Sim|0.1.2
code-spell-checker|str|3.0.1
code-spell-checker-french|str|0.4.1
svelte-vscode|sve|108.4.1
even-better-toml|tam|0.19.2
html-preview-vscode|tht|0.2.5
pypi-assistant|twi|2.0.5
sort-lines|Tyr|1.11.0
errorlens|use|3.17.0
vscode-icons|vsc|12.7.0
volar|Vue|2.0.8
gitblame|wad|10.11.0
vscode-todo-highlight|way|1.0.5
change-case|wma|1.0.0
better-align|wwm|1.1.6
pretty-ts-errors|Yoa|0.5.4

(2 theme extensions excluded)

</details>

**Comment by farfromrefug** [comment]
@daiyam yes reproduced with 1.89. Maybe i should report that in vscode repo? I found a way to reproduce easily

**Comment by daiyam** [comment]
Can you provide the following infos?  And can you confirm the last question? Thx

**Desktop (please complete the following information):**
 - OS: [e.g. Mac OS, Ubuntu 20.04, etc]
 - Architecture [e.g. x64, ia32, arm64]
 - Version [e.g. 1.33.0]
 - App Manager [e.g. Winget, Homebrew, Snap, AUR, RPM, Nix, ...]
 - Sandboxed [e.g. no, Flatpak, Snap]

**Please confirm that this problem is VSCodium-specific**
- [ ] This bug doesn't happen if I use Microsoft's Visual Studio Code. It only happens in VSCodium.

**Comment by farfromrefug** [comment]
@daiyam 
* OS: MacOs and PopOS 22.04
* Architecture: x64
* Version: 1.89.1
* App Manager: APT on Ubuntu and dmg on macos
* Sandboxed: no

I tested in vscode 1.89.1 and i cant reproduce it. 
Then i tested in vscodium using `codium --disable-extensions ` and i see the issue:
i found an easy way to reproduce it:
- start an integrated terminal
- start a search in the integrated terminal for a string which will appear in the console
- start a long processing command (like a c++ build)
- while the command is running copy a text in the clipboard different from the search terms
- go in the terminal and press enter to add a new line while the long processing command is going
- try to paste in an editor 
=> the search terms is pasted instead of the one you copied before

**Comment by markus-zhang-mistplay** [comment]
I got the same issue, closing the search box in terminal window fixed it, but would love an official fix.

**Comment by tintii** [comment]
+1 same issue, very frustrating. Would be nice to keep copy on selection but not have the search override.

**Comment by farfromrefug** [comment]
@markus-zhang-mistplay @tintii https://github.com/microsoft/vscode/issues/214092
